backend/src/app/srepl/helpers.clj

```diff
@@ -85,6 +88,46 @@
       WHERE file_id = ANY(?)
         AND id IS NOT NULL")
 
+(defn enable-team-feature!
+  [team-id feature & {:keys [debug] :or {debug false}}]
+  (when (and (not debug) (not (contains? cfeat/supported-features feature)))
+    (ex/raise :type :assertion
+              :code :feature-not-supported
+              :hint (str "feature '" feature "' not supported")))
+
+  (let [team-id (h/parse-uuid team-id)
+        feature (str feature)]
```

Comment:
This step is not necessary, I think is better to have an assertion for string than coercing all to string